statsmodels/regression/recursive_ls.py

```diff
@@ -267,6 +266,176 @@ def __init__(self, model, params, filter_results, cov_type='opg',
+              See the examples.
+            - tuple : A tuple of arrays in the form (R, q). If q is given,
+              can be either a scalar or a length p row vector.
+        loc : int, str, or datetime, optional
+            Zero-indexed observation number at which to compute the t-test.
+            Can also be a date string to parse or a datetime type. Default is
+            to compute the t-test using the full-sample estimates. Cannot be
+            used in combination with `cov_p`.
+        cov_p : array-like
+            An alternative estimate for the parameter covariance matrix.
+            If None is given, self.normalized_cov_params is used.
+        scale : float, optional
+            An optional `scale` to use.  Default is the scale specified
+            by the model fit.
+        use_t : bool, optional
+            If use_t is None, then the default of the model is used.
+            If use_t is True, then the p-values are based on the t
+            distribution.
+            If use_t is False, then the p-values are based on the normal
+            distribution.
+
+        Returns
+        -------
+        res : ContrastResults instance
+            The results for the test are attributes of this results instance.
+            The available results have the same elements as the parameter table
+            in `summary()`.
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> import statsmodels.api as sm
+        >>> data = sm.datasets.longley.load()
+        >>> data.exog = sm.add_constant(data.exog)
+        >>> results = sm.OLS(data.endog, data.exog).fit()
+        >>> r = np.zeros_like(results.params)
+        >>> r[5:] = [1,-1]
+        >>> print(r)
+        [ 0.  0.  0.  0.  0.  1. -1.]
+
+        r tests that the coefficients on the 5th and 6th independent
+        variable are the same.
+
+        >>> T_test = results.t_test(r)
+        >>> print(T_test)
+                                     Test for Constraints
+        ==============================================================================
+                         coef    std err          t      P>|t|      [0.025      0.975]
+        ------------------------------------------------------------------------------
+        c0         -1829.2026    455.391     -4.017      0.003   -2859.368    -799.037
+        ==============================================================================
+        >>> T_test.effect
+        -1829.2025687192481
+        >>> T_test.sd
+        455.39079425193762
+        >>> T_test.tvalue
+        -4.0167754636411717
+        >>> T_test.pvalue
+        0.0015163772380899498
+
+        Alternatively, you can specify the hypothesis tests using a string
+
+        >>> from statsmodels.formula.api import ols
+        >>> dta = sm.datasets.longley.load_pandas().data
+        >>> formula = 'TOTEMP ~ GNPDEFL + GNP + UNEMP + ARMED + POP + YEAR'
+        >>> results = ols(formula, dta).fit()
```

Comment:
I'm guessing this docstring is copy/pasted from elsewhere.  Can the existing version just be pinned here?  i.e. so if it changes down the road this one doesn't become inaccurate?

Reply:
Not directly since we added a parameter. There might be something to do I guess.